doc/user/layouts/shortcodes/views-indexes/table-usage-pattern-intro.html

```diff
@@ -1,8 +1,9 @@
-In Materialize, both [indexes](/concepts/indexes) on views and [materialized
-views](/concepts/views/#materialized-views) incrementally update the view
-results when Materialize ingests new data. Whereas materialized views persist
-the view results in durable storage and can be accessed across clusters, indexes
-on views compute and store view results in memory within a **single** cluster.
+In Materialize, both [indexes](/concepts/indexes) (specifically on views) and
```

Comment:
Just tweaked because "both indexes on views and materialized views ..." could be read as indexes on views and indexes on materialized views.
Thought about flipping to "both materialized views and indexes on views" ... eh.

Reply:
This wording is potentially more confusing. Left a suggestion that removes the bearing term "compute results" and just mentions that indexes _store_ the latest, incrementally updated results, which is valid for both indexes and materialized views.